packages/waku/src/router/fs-router.ts

```
@@ -149,6 +156,15 @@ export function unstable_fsRouter(
             render: 'static',
             ...config,
           });
+        } else if (pathItems.at(-1)?.startsWith('_part')) {
+          const order = parseInt(pathItems.at(-1)!.split('-')[0]!.substring(5));
+          createPagePart({
+            path,
+            component: mod.default,
+            render: 'static',
+            order,
```

I think we should throw an error if `Number.isNaN(order)`.